### PR TITLE
Add a page to explain the UIKit use

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | We need to explain a bit how the UIKit is used and more

### CODE REVIEW

File: src/content/1.7/development/uikit.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- custom-file-input
+- growl (using jquery growl)
+- ps-tags
+- spinner
+- ps-switch
+- switch-input
+- md-checkbox
+- toast
+
+## Where is the UIKit used ?
```
Comment (PierreRambaud) [inline src/content/1.7/development/uikit.md:40]:
```suggestion
## Where is the UIKit used?
```

File: src/content/1.7/development/uikit.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+It's mainly used on migrated pages, legacy pages rely on Bootstrap 3 for the moment, and it's not using the UIKit at all. The UIKit can also be used inside modules, even if they are on legacy pages, by overriding styles inside their particular markup.o
+
+## How to contribute ?
```
Comment (PierreRambaud) [inline src/content/1.7/development/uikit.md:44]:
```suggestion
## How to contribute?
```

File: src/content/1.7/development/uikit.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: The UIKit
+weight: 70
+---
+
+# What's the PrestaShop UIKit ?
```
Comment (PierreRambaud) [inline src/content/1.7/development/uikit.md:6]:
```suggestion
# What's the PrestaShop UIKit?
```

File: src/content/1.7/development/uikit.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+The UIKit is a project extending Bootstrap 4 in order to provide some components with PrestaShop's colors. You can see every components on [this page](https://cdn.rawgit.com/PrestaShop/prestashop-ui-kit/develop/index.html).
+
+## How do we use the UIKit ?
```
Comment (PierreRambaud) [inline src/content/1.7/development/uikit.md:10]:
```suggestion
## How do we use the UIKit?
```

File: src/content/1.7/development/uikit.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+- You are able to import source files, it means that you'll need to rebuild the UIKit inside your own styling architecture, but you'll be able to use every variables inside it.
+
+```
+@import "~prestakit/scss/application";
+```
+
+## Custom components
+
+Some components are not provided from Bootstrap, they have been done for our own use, here is a list of them :
```
Comment (PierreRambaud) [inline src/content/1.7/development/uikit.md:28]:
```suggestion
Some components are not provided from Bootstrap, they have been done for our own use, here is a list of them:
```

File: src/content/1.7/development/uikit.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+- ps-tags
+- spinner
+- ps-switch
+- switch-input
+- md-checkbox
+- toast
+
+## Where is the UIKit used?
+
+It's mainly used on migrated pages, legacy pages rely on Bootstrap 3 for the moment, and it's not using the UIKit at all. The UIKit can also be used inside modules, even if they are on legacy pages, by overriding styles inside their particular markup.o
```
Comment (PierreRambaud) [inline src/content/1.7/development/uikit.md:42]:
```suggestion
It's mainly used on migrated pages, legacy pages rely on Bootstrap 3 for the moment, and it's not using the UIKit at all. The UIKit can also be used inside modules, even if they are on legacy pages, by overriding styles inside their particular markup.
```

File: src/content/1.7/development/uikit.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+- ps-tags
+- spinner
+- ps-switch
+- switch-input
+- md-checkbox
+- toast
+
+## Where is the UIKit used?
+
+It's mainly used on migrated pages, legacy pages rely on Bootstrap 3 for the moment, and it's not using the UIKit at all. The UIKit can also be used inside modules, even if they are on legacy pages, by overriding styles inside their particular markup.
```
Comment (atomiix) [inline src/content/1.7/development/uikit.md:42]:
```suggestion
It's mainly used on migrated pages, legacy pages rely on Bootstrap 3 for the moment and are not using UIKit at all. UIKit can also be used inside modules, even if they are on legacy pages, by overriding styles inside their particular markup.
```

File: src/content/1.7/development/uikit.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+## How to contribute?
+
+You're able to contribute to the UIKit on [his own repository](https://github.com/PrestaShop/prestashop-ui-kit) but keep in mind that your changes won't be effective until we provide a new release on npm and your changes won't be effective on the PrestaShop project until we update the dependency inside the new-theme directory.
```
Comment (atomiix) [inline src/content/1.7/development/uikit.md:46]:
```suggestion
You can contribute to UIKit on [his own repository](https://github.com/PrestaShop/prestashop-ui-kit) but keep in mind that your changes won't be effective until we provide a new release on npm and your changes won't be effective on the PrestaShop project until we update the dependency inside the new-theme directory.
```

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @NeOMakinG

**Pull request merged**